cli/impl/src/main/java/io/helidon/build/cli/impl/InitCommand.java

```diff
@@ -119,6 +126,33 @@ protected void invoke(CommandContext context) throws Exception {
         }
     }
 
+    private String promptHelidonVersion(ArchetypesData archetypesData, boolean showLatest) {
+        String defaultHelidonVersion = defaultHelidonVersion(archetypesData);
+        List<String> versions = Lists.map(archetypesData.versions(), Version::id);
```

Comment:
The code would be easier to read if we'd encapsulate that ogic in ArchetypesData. How about having `archetypesData.rawVersions()` or maybe use `Version` here directly.